### DISCUSSION TITLE
I can't signup or login

### DISCUSSION THREAD

**Discussion by faithfulojebiyi** [vercel/community#3894] [Account Issue] [answered]
### I am not able to signup or login
The account was just newly created.

<img width="515" alt="Screenshot 2023-09-02 at 09 26 01" src="https://github.com/vercel/community/assets/26503966/3280e663-5c46-4687-9406-0143001c6d2e">
<img width="486" alt="Screenshot 2023-09-02 at 09 25 48" src="https://github.com/vercel/community/assets/26503966/57be66e4-8608-44f9-8bee-51e95ff64b4a">

**Comment by Waris-Shaik** [comment]
i am also unable to signup and i did send mail to registration@vercel.com i might get response from them soon.

**Reply by marcoacurto** [reply]
I'm in the same page, the same messages appeared to me. 
It was the first time I tried to sign up on their website.
I also sent them an email, hope they could unlock my user too.

**Comment by leerob** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
